src/lib/annotations/annotatorUtil.js

```diff
@@ -420,12 +420,10 @@ export function eventToLocationHandler(locationFunction, callback) {
             return;
         }
 
-        evt.stopPropagation();
         evt.preventDefault();
+        evt.stopPropagation();
         const location = locationFunction(evt);
-        if (location) {
```

Comment:
I found it preferable to let the callback handle an invalid location rather than avoiding a call on the callback. This is because the callback can handle/display an error rather than hiding the error overall.